src/app/anon/login.html

```diff
@@ -1,9 +1,19 @@
-<div class="background">
-	<div class="logInTile">
-	<form name="login.form" ng-submit="login.login()" novalidate>
-	<input type="email" name="email" ng-model="login.user.email" placeholder="Email Address" required />
-	<input type="password" name="password" ng-model="login.user.password" placeholder="Password" required />
-	<input type="submit" value="Log In" />
-	</form>
+<div id="logInBackground">
+	<div id="logInTile">
+		<p>Welcome! Please Log In!</p>
+			<form name="login.form" ng-submit="login.login()" novalidate>
+				<div>
```

Comment:
are the div's doing anything?
I know i told you to put them in, but it looks like you are controlling all your are styling already without needing this.  having a wrapper div is a good idea when you are going to add some common styling.  you typically would add a class to those divs.  you can keep it here, just wanted to give you some more info on this.  Did you look at the ones on bootstrap?